python/tvm/relax/frontend/pytorch_fx.py

```diff
@@ -0,0 +1,758 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+# pylint: disable=invalid-name, inconsistent-return-statements, unidiomatic-typecheck
+# pylint: disable=import-outside-toplevel
+"""PyTorch FX frontend of Relax."""
+from typing import Dict
+
+from tvm import relax
+import numpy as np  # type: ignore
+
+
+class TorchFXTranslator:
+    """A translator from PyTorch FX to Relax."""
+
+    import torch  # type: ignore
+    from torch import fx
```

Comment:
Is this import still useful? I see we are now using local import in each method.

Reply:
We need them in type annotation